docs/qupath.rst

```diff
@@ -1,69 +1,147 @@
 Analyze OMERO data using QuPath
 ===============================
 
-QuPath is a cross-platform software application designed for bioimage
-analysis - and specifically to meet the needs of whole slide image
-analysis and digital pathology.
+Description:
+------------
+
+QuPath is a cross-platform software application designed for bioimage analysis - and specifically to meet the needs of whole slide image analysis and digital pathology.
 See \ https://github.com/qupath/qupath/wiki/
 
-#.  In OMERO.web, identify an image in the idr0018 project and the
-    dataset Baz1a-14-100-gastrointestinal.
+We will show:
+
+- How to open an image from OMERO server in QuPath
+
+- How to draw annotations and perform simple Cell detection in QuPath
+
+- How to export the ROIs from QuPath as OME-XML
+
+- How to import the OME-XML to the OMERO.server and attach the QuPath ROIs to the original image in OMERO
+
+**Resources:**
+--------------
+
+- Data: example images from
+
+  - IDR data http://idr.openmicroscopy.org/webclient/?show=project-101
+
+-  Plugin ``ome-omero-roitool`` for import and export of ROIs to or from OMERO using OME-XML format
+
+   - https://github.com/glencoesoftware/ome-omero-roitool
+
+-  Scripts for export and import of ROIs from/to QuPath in OME-XML format
+
+   - https://github.com/glencoesoftware/ome-omero-roitool/tree/master/src/dist/QuPath.scripts
+
+
+Setup:
+------
+
+Download QuPath v0.2.0 or later from https://qupath.github.io/.
+
+
+**Step-by-step:**
+-----------------
+
+#. In OMERO.web, identify an image in the idr0018 project and the dataset Baz1a-14-100-gastrointestinal.
+
+#. Select the first image and double-click on it. This will open the image in OMERO.iviewer.
+
+#. In the OMERO.iviewer tab, select the whole URL in the address bar of your browser and copy it, for example using right-click and Copy.
+
+#. In QuPath, select *File > Open URL...* and paste the link into the dialog.
+
+   |image0|
+
+#. Click *OK*. In the following dialog, enter your credentials.
```

Comment:
The user will not know the idr credentials

Reply:
Opening from outreach server

Reply:
Actually, if the IDR image is opened, then no credentials are asked for - all works as is, no probs

Reply:
Explained in https://github.com/ome/omero-guide-qupath/pull/1/commits/4675e26f6d0e4346175bc15712b4560b947edc7f